src/components/ui/passwordValidationItem.tsx

```diff
@@ -0,0 +1,37 @@
+import { cn } from '@/lib/utils';
+import { CircleCheck } from 'lucide-react';
+
+interface PasswordValidationItemProps {
+  text: string;
+  isValid: boolean;
+  className?: string;
+}
+
+/**
+ * @example
+ * ```ts
+ * return (
+ * <PasswordValidationItem
+ * isValid={false}
+ * text={'Less than or equal to 8 characters long'}
+ * />
+ * )
+ * ```
+ */
+
+export function PasswordValidationItem({
+  text,
+  isValid = false,
+}: PasswordValidationItemProps) {
+  return (
+    <p className={cn('flex', { 'items-center text-sm gap-1': !isValid })}>
+      <CircleCheck
+        className={cn('stroke-white', {
+          'fill-primary': isValid,
```

Comment:
The icon color is supposed to `Primary-light (Cyan-500)`.
<img width="530" alt="image" src="https://github.com/Tascurator/tascurator-frontend/assets/144519746/72da82ad-6265-4953-815f-0b5f6789251c">